components/prism-core.js

```diff
@@ -12,13 +12,20 @@ var _self = (typeof window !== 'undefined')
  * @author Lea Verou http://lea.verou.me
  */
 
+var manual = false;
+
+if (_self.Prism) {
+	manual = _self.Prism.manual;
+}
+
 var Prism = (function(){
 
 // Private helper vars
 var lang = /\blang(?:uage)?-(\w+)\b/i;
 var uniqueId = 0;
 
 var _ = _self.Prism = {
+	manual: manual,
```

Comment:
I would be in favor of minimizing the number of changes required. It would all be done here:

```js
manual: _self.Prism && _self.Prism.manual
```

That way, there's no need for the code changes above!